Adds Helm configuration + note on having to enable the script (#14414)
DataDog/documentation: content/en/agent/guide/secrets-management.md

| Read from files        | `ENC[file@/path/to/file]`                        |
| Kubernetes Secrets     | `ENC[k8s_secret@some_namespace/some_name/a_key]` |

{{< tabs >}}
{{% tab "Helm" %}}

> To use this executable with our Helm chart, set it as follows:

Reviewer: ```suggestion
To use this executable with the Helm chart, set it as the following:
```

```
This `Role` gives access to the `Secret: database-secret` in the `Namespace: database`. The `RoleBinding` links up this permission to the `ServiceAccount: datadog-agent` in the `Namespace: default`. This needs to be manually added to your cluster with respect to your resources deployed.

> **Note:**
> - On top of the additional permissions, the script to read from multiple providers `"/readsecret_multiple_providers.sh"` must be enabled when using the Kubernetes Secrets provider.

Reviewer: ```suggestion
In addition to these permissions, you need to enable the script to read from multiple providers `"/readsecret_multiple_providers.sh"` when using the Kubernetes Secrets provider.
```
